### ISSUE TITLE
Radio Product Options - Product Page

### ISSUE THREAD

**Issue by SemperFiWS** [cubecart/v6#654]
If you assign a single radio option group to a product via an option set, all radio option groups are displayed on the product page.
Yet the name of the assigned group is used for all.

Example
![radio-options-bug-3](https://cloud.githubusercontent.com/assets/6945093/8414482/5201e510-1e66-11e5-8c17-21f6f7b11600.png)

**Comment by SemperFiWS** [comment]
This does not happen if the a single radio option group is assigned manually.

**Comment by abrookbanks** [comment]
Hi Daren, I'm pretty confident this is residual bug from https://github.com/cubecart/v6/commit/183370baf660220079ebd3df1269e1306a63c89a
Upgrading from to 6.0.6 and testing this with a new product and it should not be reproducible.

**Comment by SemperFiWS** [comment]
The customer that reported this issue is using 6.0.6 already.

You can see the issue here:
http://www.cubecartservices.com/test-optioned-product-1.html
which is also using 6.0.6.

**Comment by abrookbanks** [comment]
Did they start on 6.0.6? This issue was probably introduced in 6.0.5.

**Comment by SemperFiWS** [comment]
Both stores are clean 6.0.6 installs.

**Comment by abrookbanks** [comment]
OK. Hmmm. I’ve not been able to reproduce it.

**Comment by SemperFiWS** [comment]
Possible fix for this and #653 / #652 bugs.

File:
/admin/skins/default/templates/products.options.php

Code Before:
```         <fieldset id="add_options" class="field_select_target">
            <legend>{$LANG.catalogue.title_option_set_append}</legend>
            <div>
               <select name="add_to_set[]" class="multi" multiple="multiple" style="width: 200px; height:200px">
                  <option value="">{$LANG.form.please_select}</option>
                  {foreach from=$GROUPS item=group}
                  {if $group.type == 0}
                  <optgroup label="{$group.name}">
                     {foreach from=$group.options key=value_id item=value_name}
                     <option value="g{$group.id}-{$value_id}">{$value_name}</option>
                     {/foreach}
                  </optgroup>
                  {else}
                  <option value="{$group.id}">{$group.name}</option>
                  {/if}
                  {/foreach}
               </select>
            </div>
         </fieldset>```

Code After
```         <fieldset id="add_options" class="field_select_target">
            <legend>{$LANG.catalogue.title_option_set_append}</legend>
            <div>
               <select name="add_to_set[]" class="multi" multiple="multiple" style="width: 200px; height:200px">
                  <option value="">{$LANG.form.please_select}</option>
                  {foreach from=$GROUPS item=group}
                  {if $group.type == 0}
                  <optgroup label="{$group.name}">
                     {foreach from=$group.options key=value_id item=value_name}
                     <option value="g{$group.id}-{$value_id}">{$value_name}</option>
                     {/foreach}
                  </optgroup>
                  {else if $group.type == 4}
                  <optgroup label="{$group.name}">
                     {foreach from=$group.options key=value_id item=value_name}
                     <option value="g{$group.id}-{$value_id}">{$value_name}</option>
                     {/foreach}
                  </optgroup>
                  {else}
                  <option value="{$group.id}">{$group.name}</option>
                  {/if}
                  {/foreach}
               </select>
            </div>
         </fieldset>```

Check the rather long email I sent you regarding this for additional information.

**Comment by abrookbanks** [comment]
Thanks so much for your patience Daren. I think this is a perfect fix! From my testing it all works very well. 

I was able to reproduce the initial issue and this fixes it correctly. Thanks I really appreciate your help.
